packages/bsky/tests/views/__snapshots__/notifications.test.ts.snap

```diff
@@ -218,7 +226,7 @@ Array [
           "uri": "record(2)",
         },
       },
-      "text": "hear that",
+      "text": "hear that label_me label_me_2",
```

Comment:
looks like labels aren't making it onto this notif?

Reply:
yeah checked code & pds does labeling in listNotifications & not implemented in appview yet

Reply:
Oh man, fail!  Thanks for catching that!